src/404.html

```diff
@@ -87,17 +87,21 @@
 		<cap-nav></cap-nav>
 		<main id="main" class="l-interiorPage">
 			<header class="u-bg-gray-500 u-col-span-full">
-				<cap-page-header heading="Hmm... can't find that.">
+				<cap-page-header heading="Not Found.">
 					<p class="u-text-white u-text-serif">
-						Maybe the last borrower didn't return that link.
+						The Caselaw Access Project has updated and simplified our site to
+						provide long-term, unrestricted access to U.S. case law.
 					</p>
 					<p class="u-text-white u-text-serif">
-						If you got here through our site or an article we host, please let
-						us know at
+						As part of this process we have deactivated portions of the site,
+						potentially including the page you were looking for.
+					</p>
+					<p class="u-text-white u-text-serif">
+						If you have questions about the changes we have made, please let us
```

Comment:
```suggestion
						If you have questions about the changes we’ve made, please let us
```
Two "haves" seems odd here.